example/advection/parthinput.advection

```diff
@@ -0,0 +1,64 @@
+# ========================================================================================
+#  Athena++ astrophysical MHD code
+#  Copyright(C) 2014 James M. Stone <jmstone@princeton.edu> and other code contributors
+#  Licensed under the 3-clause BSD License, see LICENSE file for details
+# ========================================================================================
+#  (C) (or copyright) 2020. Triad National Security, LLC. All rights reserved.
+#
+#  This program was produced under U.S. Government contract 89233218CNA000001 for Los
+#  Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+#  for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+#  in the program are reserved by Triad National Security, LLC, and the U.S. Department
+#  of Energy/National Nuclear Security Administration. The Government is granted for
+#  itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+#  license in this material to reproduce, prepare derivative works, distribute copies to
+#  the public, perform publicly and display publicly, and to permit others to do so.
+# ========================================================================================
+
+<comment>
+problem = Linear advection
+
+<job>
+problem_id = advection
+
+<mesh>
+refinement = adaptive
+numlevel = 3
+
+nx1 = 64
+x1min = -0.5
+x1max = 0.5
+ix1_bc = periodic
+ox1_bc = periodic
+
+nx2 = 64
+x2min = -0.5
+x2max = 0.5
+ix2_bc = periodic
+ox2_bc = periodic
+
+nx3 = 1
+x3min = -0.5
+x3max = 0.5
+
+<meshblock>
+nx1 = 16
+nx2 = 16
+#nx3 = 1
+
+<output1>
+file_type = hdf5
+dt = 0.05
+variable = prim
```

Comment:
We should really get rid of this. It doesn't do anything, since which variables are output are set by `Metadata`.

Reply:
That's true, but this comment actually got me thinking.  It would be nice at some point to have some control over which variables had Metadata::Graphics set from the input file.  Not sure what the right model is here.  Just a list of variables, perhaps?

Reply:
I think a list of variables would suffice. We could implement this pretty easily I think... just add the graphics metadata flag to variables tagged for output.

Reply:
> We should really get rid of this. It doesn't do anything, since which variables are output are set by `Metadata`.

Can you still control which variables are written to disk through the parameter file? That seems like a very necessary feature, we don't always want the same variables written to disk for every simulation. That shouldn't be hard coded.

Reply:
Yes clearly we need control of this.  It's just a matter of setting the Metadata::Graphics flag for each variable you want to show up.  Right now this is hard coded in the examples, but only because we haven't written the little bit of code that parses some field in the input file and sets the flag.

Reply:
Made an issue: #98